src/scripts/index.js

```diff
@@ -1,10 +1,502 @@
 import '../styles/main.css';
-import Button from './Button';
-import CallToAction from './CallToAction';
+// import Button from './Button';
+// import CallToAction from './CallToAction';
 
-const button = new Button(document.querySelector('.button'));
-const cta = new CallToAction(document.querySelector('.button-cta'));
+// const button = new Button(document.querySelector('.button'));
+// const cta = new CallToAction(document.querySelector('.button-cta'));
+
+// button.registerEvents();
+// cta.registerEvents();
+// cta.changeBgColor();
+
+
+// const toDoColumn = document.querySelector('.toDo');
+
+// if (toDoColumn !== null){
+//   toDoColumn.addEventListener('dblclick', function() {
+//     console.log(toDoColumn);
+//     console.log('apelez fct openmodal');
+//     openModal();
+//   });
+
+// }
+
+const modal = document.getElementById('modal');
```

Comment:
You should encapsulate the methods and properties. It's better to use a class for this, or break down the code into smaller chunks.